app/comments/service.py

```diff
@@ -94,34 +96,39 @@ async def get_child_comments(
 
 
 async def create_comment(
-    session: AsyncSession,
-    user_id: int,
-    post_id: int,
-    comment_data: CommentsCreateRequest,
-):
-    ...
-    # depth = 0
-    # if comment_data.parent_id:
-    #     stmt = select(Comment).where(Comment.id == comment_data.parent_id)
-    #     result = await session.execute(stmt)
-    #     parent_comment = result.scalar_one_or_none()
-
-    #     if not parent_comment:
-    #         raise HTTPException(status_code=404, detail="부모 댓글을 찾을 수 없습니다.")
-
-    #     depth = parent_comment.depth + 1
-
-    # db_comment = Comment(
-    #     author_id=user_id,
-    #     post_id=post_id,
-    #     parent_id=comment_data.parent_id,
-    #     detail=comment_data.detail,
-    #     depth=depth,
-    # )
+    session: AsyncSession, author: User, comment: CommentCreate, post_id: int
+) -> CommentRead:
+    stmt = select(Post).where(Post.id == post_id)
+    result = await session.execute(stmt)
+    db_post = result.scalar_one_or_none()
```

Comment:
요거 참고로 이렇게 한줄로 쓸 수 있어용

```suggestion
    db_post = await session.scalar(stmt)
```

Reply:
오 참고하고 변경하겠습니다 ~

Reply:
이렇게 남겨주신 코드 변경 comment의 변경내용을 적용하려면 pr 내 `Commit suggestion`으로 일일이 commit을 남기는게 좋나요 아니면 일괄적으로 수정 후 commit을 남기는게 좋을까요?

Reply:
> 이렇게 남겨주신 코드 변경 comment의 변경내용을 적용하려면 pr 내 `Commit suggestion`으로 일일이 commit을 남기는게 좋나요 아니면 일괄적으로 수정 후 commit을 남기는게 좋을까요?

-> 로컬에서 한번에 작업 후 commit 하겠습니다.